### ISSUE TITLE
Changing Router wifi password stops Provision

### ISSUE THREAD

**Issue by arjunmenon** [radiodan/provision#16]
Hey
I have come to notice if my router wifi password is changed, the radiodan-provision has a hard time reconnecting to it again with the new password.

Its 12 AM here and for the past almost 20 min I have been entering, reconnecting, entering my new wifi setup in the temporary hotspot in a loop. As expected reverting to my old router password makes the Provision to work, but this no good.

Where could be the problem?

Also,if the radiodan is not successful in the first attempt, it does not show up the popup. I have to search for the gateway in the description, but some phones dont show them.

Those are the two bugs which are bugging a lot.

**Comment by andrewn** [comment]
Hi,

When you say "provision" does not reconnect, do you mean "wifi-connect"? "wifi-connect" is the thing that manages connecting to wifi networks and showing the web-based user interface if no wifi connection is found when the Pi boots up.

If so, you should try ssh'ing into your Pi and deleting the existing wifi network config. There's a command `forget-wifi-networks` that will do this. It deletes `/var/lib/connman` and `/data/connections.json` then reboots the Pi.

For the second part of your issue: 

> Also,if the radiodan is not successful in the first attempt, it does not show up the popup. I have to search for the gateway in the description, but some phones dont show them.

I'm not 100% sure what you mean by this. Some phones don't seem to trigger the captive portal page that shows the web page to select a wifi network.

**Comment by arjunmenon** [comment]
Yes the captive portal. Sometimes it never shows in the second attempt, even tested this theory on a ipad. I would always have to look for the `router` address there or in android `gateway` address.
